### PR TITLE
Set up page component docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-page-component-docs-061820](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-page-component-docs-061820)

File: src/app/docs/page/page-docs.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<sky-docs-demo-page
+  moduleName="SkyPageModule"
+  moduleSourceCodePath="src/app/public/modules/page/"
+  pageTitle="Page"
+>
+  <sky-docs-demo-page-summary>
+    The page component wraps the contents of an index.html file....
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/page/page-docs.component.html:7]: How about something like...

> The page component resets the SPA's background to white, and provides an easy way to override styling.

Depending on what we come up for design guidelines might have an impact on this sentence.

File: src/app/docs/page/page-docs.component.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  >
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/page"
+    >
+    </sky-docs-code-example>
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/page`"
+    >
+    </sky-docs-code-example> -->
+</sky-docs-demo-page>
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/page/page-docs.component.html:31]: We should slap an `@internal` tag on the `SkyPageThemeAdapterService`. Those aren't to be used by consumers.

File: src/app/docs/page/page-docs.component.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/docs/page/page-docs.component.html:7]: Looks good.

File: src/app/docs/page/page-docs.component.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/docs/page/page-docs.component.html:31]: Will do.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-page-component-docs-061820](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-page-component-docs-061820)

**Pull request merged**